agora_analytics/manifest.json

```
@@ -0,0 +1,55 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "a752523e-9c3d-458c-a91a-af0c9fae5adc",
+  "app_id": "agora-analytics",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "View Agora Analytics Collector metrics in Datadog",
+    "title": "Agora Analytics",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Agora Analytics Overview - Dashboard",
+        "image_url": "images/agora_analytics_dashboard.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Monitoring",
+      "Offering::Integration",
+      "Supported OS::Linux",
+      "Supported OS::Mac OS",
+      "Supported OS::Windows"
```

We also support a new field for adding the types of data that you submit into and query from Datadog. I believe this integration only sends in metrics, but feel free to add more if relevant:

```suggestion
      "Supported OS::Windows",
      "Submitted Data Type::Metrics"
```

---

fixed